### PR TITLE
user fixups

### PR DESCRIPTION
**Pull request by Gjum**
see commit msgs

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -32,7 +32,15 @@
                     <ul class="nav navbar-nav">
                     </ul>
                     <ul class="nav navbar-nav navbar-right">
-                        <li><a href="{% url 'users:signup' %}">Sign up</a></li>
+						{% if user %}
```
Comment (metalmatze) [inline templates/base.html:35]: You should use 4 spaces and not tabs! See [CONTRIBUTING.md](https://github.com/7Pros/circuit/blob/develop/CONTRIBUTING.md)

File: users/views.py [hunk 2]
```diff
@@ -51,7 +51,10 @@ def UserCreateConfirmView(request, token):
         user.is_active = True
         user.save()
 
-        # TODO redirect('login')
+        # TODO redirect after confirmation:
+        # to profile if logged in
```
Comment (metalmatze) [inline users/views.py:55]: Nobody will be logged in at this point.

File: users/views.py [hunk 3]
```diff
@@ -51,7 +51,10 @@ def UserCreateConfirmView(request, token):
...
+        # to login if not logged in
+        # to signup if invalid token etc
```
Comment (metalmatze) [inline users/views.py:57]: Take a look at the `except ObjectDoesNotExist` this is exactly what you have written here.

File: users/urls.py [hunk 4]
```diff
@@ -13,6 +14,7 @@
     url(r'^signup/$', UserCreateView.as_view(), name='signup'),
     url(r'^signup/confirm/(?P<token>\w+)/$', views.UserCreateConfirmView, name='signup_confirm'),
     # url(r'^login/$', UserLoginView.as_view(), name='login'),
+    # url(r'^login/$', UserLogoutView.as_view(), name='logout'),
```
Comment (metalmatze) [inline users/urls.py:17]: I've changed this to `r'^logout/$'`

File: templates/base.html [hunk 5]
```diff
@@ -32,7 +32,15 @@
...
+							<li><a href="{% url 'users:profile' pk=user.pk %}">@{{ user.username }}</a></li>
```
Comment (metalmatze) [inline templates/base.html:36]: `{% url 'users:profile' pk=user.pk %}` throws NoReverseMatch at /users/signup/

Has this been tested?

Comment (Gjum) [inline templates/base.html:36]: should be `user.is_authenticated`, then it works. was fixed on #59 branch

Comment (Gjum) [inline templates/base.html:36]: f7bdf7eeadbd7e329b51ec8e4bee3ca457a41c67 fixes this, specifically [this line](https://github.com/7Pros/circuit/commit/f7bdf7eeadbd7e329b51ec8e4bee3ca457a41c67#diff-8944b57466f08564caa53a1988261ae0L35).
You might revert 1a3d8d214b6b6b361dd41c31285f0dfae07a3101 and patch it this way.

**Pull request merged**